### DISCUSSION TITLE
SyntaxError: Unexpected reserved word

### DISCUSSION THREAD

**Discussion by etekumoses** [vitejs/vite-plugin-react#55] [Q&A]
I am trying to run a react app using npm run dev but i keep getting this error 
```
PS D:\2022\ippe> npm run dev

> ippe@0.0.0 dev
> vite

file:///D:/2022/ippe/node_modules/vite/bin/vite.js:7
    await import('source-map-support').then((r) => r.default.install())
    ^^^^^

SyntaxError: Unexpected reserved word
    at Loader.moduleStrategy (internal/modules/esm/translators.js:140:18)
    at async link (internal/modules/esm/module_job.js:42:21)
PS D:\2022\STI\System\ippe>
```

I am not sure how to fix it  even during deployment to gcp i keep getting it..

**Comment by ArnaudBarre** [comment]
It's seems to be a commonjs/esm interop issue. Top level await are not available in commonjs.

If you believe this is an issue with Vite code source, please provide a reproduction: https://antfu.me/posts/why-reproductions-are-required
